src/Microsoft.AspNetCore.Razor.Evolution/RazorEngine.cs

```diff
@@ -14,15 +15,26 @@ public static RazorEngine Create()
             return Create(configure: null);
         }
 
+        public static RazorEngine Create(bool icanhasDefaults)
```

Comment:
I'll update the baselines in the followup. The parameter goes away and is set by default.